src/uct/ib/base/ib_md.c

```diff
@@ -134,7 +134,7 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
 #endif
 
     {"PCI_BW", "",
-     "Maximum bandwidth of PCI bus connected to HCA\n",
+     "Maximum fine-grained bandwidth of PCI bus connected to HCA\n",
```

Comment:
don't know what is fine-grained.. pls write "effective throughput" or "effective data transfer rate" as in XIlinx doc

Reply:
wording updated

I will squash changes after approve